index.html

```diff
@@ -30,6 +33,170 @@ <h2>I’am a Software Developer</h2>
             </div>
         </div>
     </section>
+    <!-- the hero section ends here -->
+
+    <!-- the works section -->
+    <section>
+        <div class="works-section">
+            <div class="recentwork">
+                <h2>My Recent Works</h2>
+            </div>
+
+            <div class="workone">
+                <img src="./images/workone.png" alt="">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] [Optional] For a small change I won't ask for the required changes as this is your last option. Please use `color: #091E42; ` so It matches the design